test/verify/check-machines-dbus

```diff
@@ -188,6 +188,73 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_not_in_text("#vm-subVmTest1-boot-order", bootOrder)
         b.wait_not_present("#boot-order-tooltip")
 
+    def testDomainMemorySettings(self):
+        b = self.browser
+
+        args = self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
```

Comment:
Harmless, but we don't need these explictit `wait_present` any more. click, wait_*(), etc. all wait for presence and visibility now.

Reply:
Done